net/ddns-scripts/files/usr/lib/ddns/update_one_com.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/sh
+
+# ONE.COM DDNS SCRIPT
+# REQUIRES CURL
+# $ opkg install curl
+
+# SCRIPT BY LUGICO
+# CONTACT: main@lugico.de
+
+[ -z "$CURL" ] && [ -z "$CURL_SSL" ] && write_log 14 "one.com communication require cURL with SSL support. Please install"
+[ -z "$domain" ]   && write_log 14 "Service section not configured correctly! Missing 'domain'"
+[ -z "$username" ] && write_log 14 "Service section not configured correctly! Missing 'username'"
+[ -z "$password" ] && write_log 14 "Service section not configured correctly! Missing 'password'"
+
+write_log 0 "one.com ddns script started"
+
+local __SUBDOMAIN __MAINDOMAIN __LOGINURL __RECORDID
+local __TTL=3600
+
+__SUBDOMAIN=$(echo $domain | sed -e 's/[^\.]*\.[^\.]*$//' -e 's/\.$//' )
+__MAINDOMAIN=$(echo $domain | sed -e "s/$__SUBDOMAIN\.//" )
+
+
+# LOGGING IN
+__LOGINURL=$( $CURL \
+  -RsSL \
+  --stderr $ERRFILE \
+  -c /tmp/one_com_cookiejar \
+  "https://www.one.com/admin/" \
+  | grep 'Login-form login autofill' \
+  | sed -e 's/.*action="//' -e 's/".*//' -e 's/\&amp;/\&/g' \
+)
+
+if ! [ "$( $CURL \
+  -RsSL \
+  --stderr $ERRFILE \
+  -c /tmp/one_com_cookiejar \
+  -b /tmp/one_com_cookiejar \
+  "$__LOGINURL" \
+  -H "Content-Type: application/x-www-form-urlencoded" \
+  -X POST \
+  -d "username=$username&password=$password&credentialId=" \
```

Comment:
This line should be change to `-d "username=${username}&password=${password}&credentialId="`
so the variable is enclosed with breaks `{` `}`